app/src/main/java/com/duckduckgo/app/browser/webview/MaliciousSiteBlockerWebViewIntegration.kt

```diff
@@ -129,7 +131,7 @@ class RealMaliciousSiteBlockerWebViewIntegration @Inject constructor(
         data class Safe(val isForMainFrame: Boolean) : IsMaliciousViewData()
         data object WaitForConfirmation : IsMaliciousViewData()
         data object Ignored : IsMaliciousViewData()
-        data class MaliciousSite(val url: Uri, val feed: Feed, val exempted: Boolean) : IsMaliciousViewData()
+        data class MaliciousSite(val url: Uri, val feed: Feed, val exempted: Boolean, val clientSideHit: Boolean) : IsMaliciousViewData()
```

Comment:
Can we leave a comment about what clientSideHit represents? That can help here. I've tried but can't tell. I believe is maybe to represent local (sync) vs remote detections(async) (?), but `false` seems to represent local which I'm confused, so not sure.

Reply:
You're right, for some reason I thought clientSideHit true should represent server hits 😅